src/geo/transform.js

```diff
@@ -314,11 +314,18 @@ class Transform {
         return this.tileSize * this.scale;
     }
 
+    // $FlowFixMe This getter returns the incorrect value. See free_camera.getDistanceToElevation for the rationale.
```

Comment:
Nit: `$FlowFixMe` suggests a flow error, which I don't think is occurring here. Should we cut it or replace with "FIXME:" ?
```suggestion
    // This getter returns the incorrect value. See free_camera.getDistanceToElevation for the rationale.
```

Reply:
I originally added "TODO" but lint was complaining (too strict perhaps?), I think I kind of abused the `$FlowFixMe` in this case. Will remove it altogether.